docs/content/user-guide/advanced-configuration.md

```
@@ -4,15 +4,30 @@ For advanced users who prefer direct file editing or need to perform bulk operat
 
 ## File Location
 
-Smart list files are stored in the Jellyfin data directory:
+Smart lists are stored in the Jellyfin data directory with the following structure:
 
 ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)